pkg/util/containers/pause.go

```diff
@@ -22,5 +24,17 @@ func IsPauseContainer(labels map[string]string) bool {
 	// they only have io.kubernetes.pod.name
 	// See https://github.com/containerd/cri/issues/922
 	_, podFound := labels[podNameLabel]
-	return !ctrFound && podFound
+	isPauseContainerByLabels := !ctrFound && podFound
+
+	// Sandbox containers are pause containers in CRI
+	// Ref:
+	// - https://github.com/containerd/cri/blob/release/1.4/pkg/server/helpers.go#L74
+	isPauseContainerBySandbox := labels[sandboxLabelKey] == sandboxLabelValue
```

Comment:
if can be good to first check if the `sandboxLabelKey` entry is present in the `labels` before, something like

```suggestion
	if val, ok :=  labels[sandboxLabelKey]; ok && val == sandboxLabelValue {
	   return true
	}
```